3p/integration.js

```diff
@@ -150,6 +150,7 @@ import {insticator} from '#ads/vendors/insticator';
 import {invibes} from '#ads/vendors/invibes';
 import {iprom} from '#ads/vendors/iprom';
 import {ix} from '#ads/vendors/ix';
+import {jioads} from '#ads/vendors/jioads';
```

Comment:
Please remove changes to this file. It is not needed for new integrations.

Reply:
If I remove "import {jioads} from '#ads/vendors/jioads'" from 3p/integration.js I am receiving the below error in ci/circleci: Checks — Your tests failed on CircleCI


[04:36:19] Using task file project/amp.js
[04:36:19] Starting 'lint'...
[04:36:23] 3p/integration.js
  440:20  error  'jioads' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)

Reply:
@jioads `register('jioads', jioads);` also needs to be removed.